cli/cmd/sync_v3.go

```diff
@@ -425,6 +426,16 @@ func syncConnectionV3(ctx context.Context, source v3source, destinations []v3des
 		Str("result", msg).
 		Msg("Sync summary")
 
+	analytics.TrackSyncFinished(ctx, invocationUUID, analytics.SyncFinishedEvent{
+		Source:        sourceSpec,
+		Destinations:  destinationSpecs,
+		Errors:        totals.Errors,
+		Warnings:      totals.Warnings,
+		Duration:      syncTimeTook,
+		Result:        msg,
```

Comment:
This would be better to have as simple "Succeeded" and "Failed". On cloud syncs, we have Status, and StatusReason.